src/frontend/js/components/Banner/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react';
+
+export enum BannerType {
+  ERROR = 'error',
+  INFO = 'info',
+  SUCCESS = 'success',
+  WARNING = 'warning',
+}
+
+interface BannerProps {
+  message: string;
+  type: BannerType;
+  rounded?: boolean;
+}
+
+const Banner = ({ message, rounded, type }: BannerProps) => {
+  const className = useMemo(
```

Comment:
Maybe have a default of info and not rounded?

Reply:
Yes good idea to use `info` as default type prop.